book_class.rb

```diff
@@ -0,0 +1,8 @@
+class Book
+  attr_accessor :title, :author
+
+  def initialize(title, author)
+    @title = title
+    @author = author
+  end
+end
```

Comment:
- [ ] Also, for the books, we need to keep a list of `rentals` and add every rental that is made to its rental list. Please create a rental array and add it to the `attr_accessor` list, besides that, create a `method` to add a new rental to the rentals list. 

**I notice you implement this feature with the `Person` class, by adding rental to the book while adding a person, this approach it closely coupled and I would strongly advise not to follow it. The `book` class should soly be responsible for maintaining association between the `book and rental`.**